www/.eslintrc

```diff
@@ -0,0 +1,18 @@
+---
+env:
+  es6: true
+  jquery: true
+  browser: true
+
+extends: eslint:recommended
```

Comment:
Don't extend so that this file will inherit from the root directory. This entire file should be simply:
```yaml
---
env:
  jquery: true
  browser: true

rules:
  strict: [2, "function"]
...
```